Source/Swig/symbol.c

```diff
@@ -382,6 +382,26 @@ String *Swig_symbol_qualified_language_scopename(Symtab *n) {
   return result;
 }
 
+/* -----------------------------------------------------------------------------
+ * Swig_symbol_add_using()
+ * ----------------------------------------------------------------------------- */
+
+void Swig_symbol_add_using(String * name, String * uname, Node * n) {
+  Hash *h;
+  h = Swig_symbol_clookup(uname,0);
```

Comment:
For consistency I'd put a space after the `,` here...

Reply:
done